FswStressTests/Scenario07_ConcurrentWatchers.cs

```diff
@@ -0,0 +1,67 @@
+namespace FswStressTests;
+
+/// <summary>
+/// Scenario 7: Concurrent watchers on same directory
+/// Tests multiple FileSystemWatcher instances monitoring the same directory
+/// </summary>
+public class Scenario07_ConcurrentWatchers : IStressScenario
+{
+    public string Name => "Concurrent Watchers";
+
+    public async Task RunAsync()
+    {
+        var testDir = Path.Combine(Path.GetTempPath(), $"fsw_test_{Guid.NewGuid()}");
+        Directory.CreateDirectory(testDir);
+
+        try
+        {
+            var watcher1Events = 0;
+            var watcher2Events = 0;
+            var watcher3Events = 0;
+
+            using var watcher1 = new FileSystemWatcher(testDir)
+            {
+                NotifyFilter = NotifyFilters.FileName,
+                EnableRaisingEvents = true
+            };
+
+            using var watcher2 = new FileSystemWatcher(testDir)
+            {
+                NotifyFilter = NotifyFilters.FileName,
+                EnableRaisingEvents = true
+            };
+
+            using var watcher3 = new FileSystemWatcher(testDir)
+            {
+                NotifyFilter = NotifyFilters.FileName,
+                EnableRaisingEvents = true
+            };
+
+            watcher1.Created += (s, e) => Interlocked.Increment(ref watcher1Events);
+            watcher2.Created += (s, e) => Interlocked.Increment(ref watcher2Events);
+            watcher3.Created += (s, e) => Interlocked.Increment(ref watcher3Events);
+
+            // Create test files
+            for (int i = 0; i < 5; i++)
+            {
+                var filePath = Path.Combine(testDir, $"test_{i}.txt");
+                await File.WriteAllTextAsync(filePath, $"Content {i}");
+                await Task.Delay(100);
```

Comment:
You don't need to await after every operation if you later wait for all events to be processed anyway.
```suggestion
```